### ISSUE TITLE
Add Voyager to a non-spring boot application 

### ISSUE THREAD

**Issue by chih-chen** [graphql-java-kickstart/graphql-spring-boot#172]
Hi, 
I'm trying to add Voyager into a non-spring boot application. The application uses spring context for DI and uses plain graphql-java. Is it possible? If yes, is there any tutorial for this? 

Thanks for any help!

**Comment by oliemansm** [comment]
There's no tutorial for this, but I'd suggest to simply take a look at the code in this project to see how voyager is included here. Looks pretty straightforward. So you could possibly just copy paste some of that stuff over to your own project to include it at your end.

**Comment by chih-chen** [comment]
What I did:
- Serve static files listed in this project
- Create controller, which contains basically the same code from [here](https://github.com/graphql-java-kickstart/graphql-spring-boot/blob/master/voyager-spring-boot-autoconfigure/src/main/java/com/oembedler/moon/voyager/boot/VoyagerController.java).

It works and the front-end page loads fine, however it shows `transmitting...` forever.
By inspecting the web page, there are some errors related to graphQL schema:

`TypeError: Cannot read property '__schema' of undefined`

I believe that the reason why this error is happening is because Voyager can't get the introspection of the schema. The question is, how Voyager will get the introspection of my schema? It makes a request to the endpoint I configured as `graphqlEndpoint`?
